.github/workflows/python-app.yml

```diff
@@ -6,44 +6,46 @@ name: Robyn Python application
 
 on:
   push:
-    branches: [ "robynpy_release" ]
+    branches: ['robynpy_release']
   pull_request:
-    branches: [ "robynpy_release" ]
+    branches: ['robynpy_release']
 
 permissions:
   contents: read
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
-          matrix:
-            python-version: ["pypy3.9", "3.9", "3.10"]
-        
+      matrix:
+        python-version: ['pypy3.9', '3.9', '3.10']
```

Comment:
github action build for pypy3.9 fails. There is specific reason we put this here so try updating it to pypy3.10 if it solves the issue.

Reply:
deleted for now